### ISSUE TITLE
No login prompt on serial 

### ISSUE THREAD

**Issue by meisteg** [agherzan/meta-raspberrypi#1098]
I have set the `cmdline.txt` and `config.txt` so I see serial prints from kernel initialization via UART. However, I do not see a login prompt on serial, only on the HDMI screen. How do I get the login prompt on serial as well?

target: raspberrypi3-64
image: core-image-base

cmdline.txt:
```
dwc_otg.lpm_enable=0 console=serial0,115200 console=ttyS0 root=/dev/mmcblk0p2 rootfstype=ext4 rootwait
```

config.txt includes:
```
enable_uart=1
```

**Comment by kraj** [comment]
set 
```
ENABLE_UART = "1"
```
in `local.conf`

**Comment by meisteg** [comment]
Not knowing better, I did originally build without `ENABLE_UART` being set and of course had no serial. After googling the issue, I learned about the `ENABLE_UART` flag. Trying to avoid another lengthy build and save time, I looked in this repo as to what that flag did, and I only saw the `config.txt` and `cmdline.txt` file changes, so I manually made them myself. I got serial output during boot, but of course no login prompt.

I went ahead and rebuilt with `ENABLE_UART` set in `local.conf`, and now I do get the login prompt on the serial.

Clearly I missed something that flag does. Does poky also do something with that flag? I'd like to learn to avoid issues in the future.

Thanks for your help!
